### PR TITLE
feat(wallet): add mock methods

### PR DESCRIPTION
**Pull request by anler**
Added methods:

- Create VTT `createVtt`
- Create Data Request `createDataRequest`
- Send Transaction `sendTransaction`

### CODE REVIEW

**Comment by Tommytrg** [comment]
@tmpolaczyk  @lrubiorod which params should I pass to these methods?

**Comment by lrubiorod** [comment]
> @tmpolaczyk @lrubiorod which params should I pass to these methods?

When we receive the info from the JsonRpcServer we have two methods, `build_data_request` and `build_value_transfer` (line 417 from /node/src/actors/json_rpc/json_rpc_methods.rs), with the messages `BuildDrt` and `BuildVtt` (line 146 from /node/src/actors/messages.rs) which they need a `Vec<ValueTransferOutput>` and the fee, or the `DataRequestOutput` and the fee.

These messages will be handled in `ChainManager` (line 146 from /node/src/actors/chain_manager/handlers.rs) which will call to methods `build_vtt` and `build_drt` (line 73 from /node/src/actors/chain_manager/transaction_factory.rs). These two methods must be changed later by the wallet logic to specify the utxos to use, now is using the first ones, and it only allow to produce one transaction by epoch because double spending

File: wallet/src/actors/app/handlers/send_transaction.rs [hunk 1]
```diff
@@ -0,0 +1,20 @@
+use actix::prelude::*;
+
+use crate::actors::App;
+use crate::api;
+
+impl Message for api::SendTransactionRequest {
+    type Result = Result<api::CreateDataReqResponse, api::Error>;
+}
+
+impl Handler<api::SendTransactionRequest> for App {
```
[the remark this answers is not in the document]
Comment (Tommytrg) [reply wallet/src/actors/app/handlers/send_transaction.rs]: We want to be able to send data to the wallet to create a transaction and show it to the user. Then, the user has to explicitly send it.

**Comment by Tommytrg** [comment]
Then, I propose the following signatures:
- createVtt(address: String, label: String, amount: Number, fee: Number) -> Transaction
- createDataRequest(rad_request: RadRequest, fee: Number) -> Transaction
- sendTransaction(transaction: Transaction) -> Boolean

**Comment by anler** [comment]
I'm closing this PR since it was intended for my master branch, not witnet's

**Pull request closed without merging**